Framework/PythonInterface/test/python/plugins/algorithms/CalculateSampleTransmissionTest.py

```diff
@@ -13,16 +13,16 @@ def test_sample_transmission_calculation(self):
 
         # Using water sample
         formula = "H2-O"
-        density = 0.033424
+        density = 0.100272
```

Comment:
The docs for the algorithm say that number density is in `Atoms / Angstrom^3`, so `1 g /cc` (standard water density) in `test_mass_density` should correspond to `0.100272 Atoms / Angstrom^3` (`3 x 0.033424 molecules / Angstrom^3`) here.

I think the point of this test was to show that mass density and number density, at standard density, give the same results. Hence why I thought changing this made more sense than changing the mass density.